qiskit/pulse/instruction_schedule_map.py

```diff
@@ -259,6 +271,54 @@ def cmd_qubits(self, cmd_name: str) -> List[Union[int, Tuple[int]]]:
                       DeprecationWarning)
         return self.qubits_with_instruction(cmd_name)
 
+    def get_signature(self,
+                      instruction: str,
+                      qubits: Union[int, Iterable[int]]
+                      ) -> Optional[inspect.Signature]:
```

Comment:
```suggestion
                      qubits: Union[int, Iterable[int]]) -> Optional[inspect.Signature]:
```

Reply:
I prefer it the way is I think? Is there any standard for this?